scripts/bls/eigen/register.sh

```diff
@@ -38,6 +48,21 @@ ensure_balance "$OP_ADDR"
 echo "Operator address: $OP_ADDR"
 echo "Delegate amount: $WAVS_DELEGATE_AMOUNT"
 
-# Register operator
 cd contracts || handle_error "Failed to change to contracts directory"
-forge script script/eigenlayer/bls/WavsRegisterOperator.s.sol -vvv --rpc-url "$RPC_URL" --private-key "$OPERATOR_KEY" --broadcast --skip-simulation --slow || handle_error "Failed to register operator"
+
+# Deposit into strategy
+forge script script/eigenlayer/bls/WavsDepositIntoStrategy.s.sol -vvv --rpc-url "$RPC_URL" --private-key "$OPERATOR_KEY" --broadcast || handle_error "Failed to deposit into strategy"
+
+# Wait based on environment
```

Comment:
Ah, very cool.

Run one script, wait the time in bash, then run the second part.
Good approach